packages/node_modules/node-red/settings.js

```diff
@@ -139,14 +139,25 @@ module.exports = {
     // The following property can be used to enable HTTPS
     // See http://nodejs.org/api/https.html#https_https_createserver_options_requestlistener
     // for details on its contents.
-    // See the comment at the top of this file on how to load the `fs` module used by
-    // this setting.
-    //
+    // See the comment at the top of this file on how to load the `fs` module used by this setting.
+    // This property can be an object, containing both a (private) key and a (public) certificate:
     //https: {
-    //    key: fs.readFileSync('privatekey.pem'),
-    //    cert: fs.readFileSync('certificate.pem')
+    //  key: fs.readFileSync('privkey.pem'),
+    //  cert: fs.readFileSync('cert.pem')
+    //},
+    // This property can also be a function (e.g. to automatic refresh the https settings):
+    //https: function() {
+    //     return {
+    //         key: fs.readFileSync('privkey.pem'),
+    //         cert: fs.readFileSync('cert.pem')
+    //     }
     //},
 
+    // The following property can be used to refresh the https settings at regular time intervals (seconds).
+    // Prerequisite: the 'https' property should be enabled (based on a function)!
+    // Caution: NodeJs version 11 or above is required to use this option!
+    //httpsRefreshInterval : 3600,
```

Comment:
Is this the setting you are looking for ?
All other settings are in mS - so this should be consistent.

Reply:
I had used seconds because updating keystores with milliseconds as units, seemed a bit silly to me at the time being.  But indeed now it might be less confusing for users...

Reply:
I'm on the side of keeping it as seconds because the idea of this needing millisecond resolution makes no sense. As long as the comment is clear what units it uses then the fact other little-used settings use ms is probably not such a big deal.

Reply:
I have changed it back to use seconds.